bin/lint-staged.js

```diff
@@ -86,6 +88,16 @@ const options = {
 
 debug('Options parsed from command-line:', options)
 
+if (options.configPath === '-') {
```

Comment:
Isn't the separator typically two dashes instead of only one?

Reply:
The standard convention for is a single dash to represent `stdin` in place of a filename.

> [**Guideline 13:**
For utilities that use operands to represent files to be opened for either reading or writing, the '-' operand should be used to mean only standard input (or standard output when it is clear from context that an output file is being specified) or a file named -.](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html)

Node uses it for it's first option instead of a filename:

>Options:
>&nbsp;&nbsp;&nbsp;\-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; script read from stdin (default if no file name is provided, interactive mode if a tty)

Reply:
👍